addons/sale_mrp/sale_mrp.py

```diff
@@ -18,7 +18,7 @@ def get_parent_move(move):
                 return get_parent_move(move.move_dest_id)
             return move
         for production in self:
-            move = get_parent_move(production.move_finished_ids[0])
+            move = production.move_finished_ids and get_parent_move(production.move_finished_ids[0]) or self.env['stock.move']
```

Comment:
I think this would work: `move = get_parent_move(production.move_finished_ids[:1])`